src/Model/GroundWaterFlow/submodules/sm-gwf-sfr-constant.f90

```diff
@@ -0,0 +1,45 @@
+submodule(SfrModule) sm_gwf_sfr_constant
+contains
+
+  !> @brief Method for solving for standard reaches
+    !!
+    !! Method to solve the continuity equation for a SFR package
+    !! reach using the standard approach.
```

Comment:
Is this routine description correct?  Seems like it should describe how we solve for constant stage reaches?